docs/toc.yml

```diff
@@ -378,6 +378,9 @@ items:
   - name: Create hosting integrations
     displayName: resources,extensibility,hosting,integrations
     href: extensibility/custom-hosting-integration.md
+  - name: Create non-container custom resources
```

Comment:
The original three articles here were delivered as a set, continuing on the previous content. Adding this one here makes this set disjointed, instead it might be better to have this under the Custom resources area in the TOC.